packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -15,11 +15,13 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
   protected db: Db;
   protected collectionName: string;
   protected tableMap: ModelTableMap;
+  protected fieldTransformMap: FieldTransformMap;
 
   public constructor(baseType: GraphQLObjectType, db: any) {
     this.db = db;
     this.tableMap = buildModelTableMap(baseType);
     this.collectionName = this.tableMap.tableName;
+    this.fieldTransformMap = getFieldTransformations(baseType);
```

Comment:
Because of how the runtime is setup:

https://github.com/aerogear/OpenVolunteerPlatform/blob/master/platform/server/src/graphql.ts#L29-L34

We might actually miss the `createdAt` and `updatedAt` fields which are automatically added when we encounter `@versioned` marker, since this (transformer retrival) happens before the `SchemaPlugin`, specifically this line https://github.com/aerogear/graphback/pull/1301/files#diff-bb76265ef553643b897bd0eb1ab72edfR292 is called.

One of the problem we could be having is actually missing the `createdAt` and `updatedAt` fields when an entity is created or updated since we do not have the appropriate transformers. 

See https://github.com/aerogear/OpenVolunteerPlatform/pull/93#discussion_r434476757 for details. 

/cc @ssd71 @wtrocki @craicoverflow

Reply:
Generally, this PR went to the very complex form of us utilizing 3 different markers etc. I would personally prefer only a single marker (versioned). It is hard to understand the implications and transformation phase it redundant - we can actually follow @machi1990 original comment and use only versioned annotation everywhere. 
Especially now when we do not have physical schema file so all transformations that will be added will not be visible to users unless they inspect schema in playground.